js/npm/etherscan/test/smoke.spec.js

```diff
@@ -14,21 +14,8 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
-import 'babel-polyfill/dist/polyfill.js';
-import es6Promise from 'es6-promise';
-es6Promise.polyfill();
+const assert = require('assert');
 
-const isNode = typeof global !== 'undefined' && typeof global !== 'undefined';
-const isBrowser = typeof self !== 'undefined' && typeof self.window !== 'undefined';
-
-if (isBrowser) {
-  require('whatwg-fetch');
-}
-
-if (isNode) {
-  global.fetch = require('node-fetch');
-}
-
-import ShapeShift from './3rdparty/shapeshift';
-
-export default ShapeShift;
+assert.doesNotThrow(() => {
```

Comment:
I am really not sure why we are doing it this way around. I would strongly suggest using `describe` and `it` blocks to be consistent with the rest of the codebase instead of using clever helpers.

```
describe('@parity.js/etherscan', () => {
  it('load the built library', () => {
    expect(() => {
      const etherscan = require('..');
    }).to.not.throw;
  });
});
```

works as well, is consistent and readable by all.